### ISSUE TITLE
Generate XML Documentation

### ISSUE THREAD

**Issue by alfonsogarciacaro** [dotnet/netcorecli-fsc#66]
1.0.0-rc3-004530
FSharp.NET.Sdk/1.0.0-beta-050000

I cannot generate XML documentation when building the project, I tried both adding the following to `.fsproj` properties:

```xml
    <DocumentationFile>$(OutputPath)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
```

Or passing it through the CLI: `dotnet build /p:DocumentationFile:Fable.Core.xml`.

It works with a C# project so apparently it's not a dotnet SDK issue. Also, doc generation seemed to work in F# for preview2.

> **Edit**: I just noticed that the build created an empty `$(OutputPath)\$(TargetFramework)` folder in my project directory (right as written, that is, without replacing the macros with the proper values).

**Comment by enricosada** [comment]
fixed by @neoeinstein 's PR #71

**Comment by alfonsogarciacaro** [comment]
Fantastic, thanks @neoeinstein!
